src/components/common/EthHashInfo/index.tsx

```diff
@@ -88,9 +96,16 @@ const PrefixedEthHashInfo = ({
   const addressBook = useAddressBook()
 
   const name = showName ? props.name || addressBook[props.address] : undefined
-  const prefix = props.address.length === 42 && settings.shortName.show ? chain?.shortName : undefined
 
-  return <EthHashInfo prefix={prefix} {...props} name={name} />
+  return (
+    <EthHashInfo
+      prefix={chain?.shortName}
+      showPrefix={settings.shortName.show}
+      copyPrefix={settings.shortName.copy}
```

Comment:
We are passing these `settings` as props because eventually the `<EthHashInfo />` will come from a component library, right?

Reply:
Exactly. I separated the local settings from the component.

Reply:
Super!